src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,84 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import React from 'react';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const pagesAmount = Math.ceil(total / perPage);
+  const pages: number[] = [];
+  const isPageFirst = currentPage === 1;
+  const isPageLast = currentPage === pagesAmount;
+
+  for (let i = 1; i <= pagesAmount; i += 1) {
+    pages.push(i);
+  }
+
+  const handlePrevPage = () => {
+    if (!isPageFirst) {
+      onPageChange(currentPage - 1);
+    }
+  };
+
+  const handlelastPage = () => {
```

Comment:
Function name should be 'handleNextPage' instead of 'handlelastPage' to better describe its purpose.